packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -1041,7 +1040,7 @@ describe('<ButtonBase />', () => {
       expect(() => {
         PropTypes.checkPropTypes(
           // @ts-ignore `Naked` is internal
```

Comment:
Can we remove that now? If not we should updated the comment (and move to `ts-expect-error`).

Reply:
There is still an error, replaced it with `@ts-expect-error` - https://github.com/mui-org/material-ui/pull/24100/commits/162367f917a7462091258451b31d12170df3ba33

Should we include by default at least React's statics in the typings?

Reply:
Library consumers shouldn't use this properties anyway. Reconciling .propTypes with the props interfaces would add quite a bit of type-related work.

Reply:
Alright, should we then leave it as is and resolve this?